data/seeds/001_users.js

```diff
@@ -0,0 +1,29 @@
+const faker = require('faker');
+const bc = require('bcryptjs');
+
+const users = [...new Array(5)].map((i, idx) => {
+  var firstName = faker.name.firstName();
+  var lastName = faker.name.lastName();
+  return {
+    email: idx == 0 ? 'lambda@labs.com' : faker.internet.email(),
+    first_name: idx == 0 ? 'Lambda' : `${firstName}`,
+    last_name: idx == 0 ? 'School' : `${lastName}`,
+    school: `Lambda School`,
+    bg_username: idx == 0 ? 'Lambda-School' : `${firstName}-${lastName}`,
+    phone: `${faker.phone.phoneNumber()}`,
+    password: bc.hashSync(
+      idx == 0 ? 'password' : `${firstName}${faker.random * 500}`,
```

Comment:
Are you seeding with a random pass here? And if so, how will you be able to use this user afterwards?